ms365_accountcreator/translations/de/LC_MESSAGES/messages.po

```diff
@@ -101,13 +111,18 @@ msgid ""
 "I consent that my st****** email is saved to prevent duplicate Account "
 "creation."
 msgstr ""
-"Ich stimme zu, dass meine st****** E-Mail Adresse gespeichert wird um zu "
-"verhindern, das mehrere Accounts für einen Student angelegt werden "
-"können."
+"Ich stimme zu, dass meine eingegebene st****** E-Mail Adresse verarbeitet"
+" und gespeichert wird um das generierte Passwort zu verschicken und um "
+"sicherzustellen, dass maximal ein Account pro Studi angelegt wird. Mir "
+"ist bewusst, dass ich diese Zustimmung widerrufen und die Löschung der "
+"Daten veranlassen kann."
 
 #: ms365_accountcreator/templates/index.html:58
 msgid "Detailed legal text email"
-msgstr "TODO Detailed legal text email"
+msgstr ""
+"Die Email Addresse wird ausschließlich auf Servern Fachbereichs und/oder "
```

Comment:
Der Zweite Zweck (Zugangsdaten für Account versenden) fehlt noch in dem Text.

Reply:
Hm. Naja, wenn das dann vom ZDI Mailserver versendet wird stimmt es ja wieder. Dann reicht das ja eigentlich. Oder?

Reply:
Ja, aber ist denke ich trotzdem schöner wenn wir alle Zwecke richtig nennen. Und es ist ja nur ein Satz mehr.

Reply:
Siehe mein suggested change.